spec/unit/classes/database/postgresql_spec.rb

```diff
@@ -63,7 +63,7 @@
             grant:    'all',
             port:     params[:database_port].to_i,
             encoding: 'UTF8',
-            locale:   'en_US.UTF-8',
+            locale:   params[:postgresql_locale],
```

Comment:
`postgresql_locale` does not appear in the `params` definition, so I would expect CI to fail, let's see…